### PR TITLE
Revert "use dart default line length (80 chars)"

### PR DESCRIPTION
**Pull request by 4ndrey**
Reverts bmw-tech/widget_driver#82

We decided to stick to the 120 line length and sacrifice 10 points in our pub.dev scores, since with default value readability became worse.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
-  <code_scheme name="Project" version="1">
-    <codeStyleSettings language="Dart">
-      <option name="RIGHT_MARGIN" value="80" />
```
Comment (robert-northmind) [inline .idea/codeStyles/Project.xml:4]: does it makes sense to keep these maybe? But just change the value to `120`?
That way, if I use some other local setting on my machine, then I still get the project specific settings in this project thanks to these?

**Review by robert-northmind — COMMENTED** [review]
Should we also make a not in changelogs? And bump version numbers? Or that is not needed?

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (4ndrey) [reply .idea/codeStyles/Project.xml:4]: good idea, I will bring it back

**Comment by 4ndrey** [comment]
> Should we also make a not in changelogs? And bump version numbers? Or that is not needed?

it's not needed from my POV

**Review by dominik089 — APPROVED** [review]

**Review by robert-northmind — APPROVED** [review]

**Pull request merged**